requirements/base.in

```diff
@@ -8,3 +8,4 @@
 -e git+https://github.com/onaio/floip-py.git@3c980eb184069ae7c3c9136b18441978237cd41d#egg=pyfloip
 -e git+https://github.com/onaio/python-json2xlsclient.git@62b4645f7b4f2684421a13ce98da0331a9dd66a0#egg=python-json2xlsclient
 -e git+https://github.com/onaio/oauth2client.git@75dfdee77fb640ae30469145c66440571dfeae5c#egg=oauth2client
+-e git+https://github.com/onaio/ona-oidc.git#egg=ona-oidc
```

Comment:
We should probably tag this to a particular commit version; Just in case a breaking change or something of the sort is introduced in the future

Reply:
sounds good.
Made this change